### PR TITLE
GH-16104: 3.46.0.1 Release Notes

### PR DESCRIPTION
**Pull request by hannah-tillman**
#16104

W/ 43 currently done

without issues:
- 15994 (added pyunit)
- 15980 (timeout increase)
- 15830 (release update)
- 16098 (pyunit fix)
- 16050 (release update)
- 16019 (UpliftDRF testing update)
- 15909 (jetty ssl init fix)
- 6788 (XGB custom metric dupe)

### CODE REVIEW

**Comment by valenad1** [comment]
@hannah-tillman Thank you, can you please add also https://github.com/h2oai/h2o-3/issues/16102 to Security?

File: Changes.md [hunk 1]
```diff
@@ -2,6 +2,56 @@
...
+Download at: <a href='http://h2o-release.s3.amazonaws.com/h2o/rel-3.46.0/1/index.html'>http://h2o-release.s3.amazonaws.com/h2o/rel-3.46.0/1/index.html</a>
+
+#### Bug Fix
+- [[#16079]](https://github.com/h2oai/h2o-3/issues/16079) - Updated warning for multithreading in `H2OFrame.as_data_frame`.
+- [[#16063]](https://github.com/h2oai/h2o-3/issues/16063) - Added error to explain method explaining incompatibility with UpliftDRF models.
+- [[#16052]](https://github.com/h2oai/h2o-3/issues/16052) - Fixed finding best split point for UpliftDRF.
+- [[#16043]](https://github.com/h2oai/h2o-3/issues/16043) - Fixed `isin()`.
+- [[#16036]](https://github.com/h2oai/h2o-3/issues/16036) - Fixed `AstMatch` failing with multinode.
+- [[#15978]](https://github.com/h2oai/h2o-3/issues/15978) - Fixed Deep Learning Autoencoder MOJO PredictCSV failure.
+- [[#15963]](https://github.com/h2oai/h2o-3/issues/15963) - Added R Tweedie packaged to test images.
```
Comment (valenad1) [inline Changes.md:16]: Its internal task - important task, but no need to be in the public list

File: Changes.md [hunk 2]
```diff
@@ -2,6 +2,56 @@
...
+- [[#15684]](https://github.com/h2oai/h2o-3/issues/15684) - Reconfigured all logs to standard error for level `ERROR` and `FATAL`.
+- [[#7325]](https://github.com/h2oai/h2o-3/issues/7325) - Implemented prediction consistency check for constrained models. 
+
+#### New Feature
+- [[#15993]](https://github.com/h2oai/h2o-3/issues/15993) - Added `custom_metric` as a hyperparameter for grid search.
+- [[#15967]](https://github.com/h2oai/h2o-3/issues/15967) - Added custom metrics for XGBoost.
+- [[#15858]](https://github.com/h2oai/h2o-3/issues/15858) - Implemented consistent mechanism that protects frames and their vecs from autodeletion.
+- [[#15857]](https://github.com/h2oai/h2o-3/issues/15857) - Integrated Target Encoder into AutoML as a pipeline transformer.
+- [[#15856]](https://github.com/h2oai/h2o-3/issues/15856) - Implemented pipeline support for grid search.
+- [[#15855]](https://github.com/h2oai/h2o-3/issues/15855) - Implemented AutoML core pipeline API.
```
Comment (valenad1) [inline Changes.md:35]: Note: remove those

File: Changes.md [hunk 3]
```diff
@@ -2,6 +2,56 @@
 
 ## H2O
 
+### 3.46.0.1 - 3/7/2024
```
Comment (valenad1) [inline Changes.md:5]: Note: update time when release is happen

**Review by valenad1 — APPROVED** [review]
LGTM: note for myself. Check the date

**Review by mn-mikke — APPROVED** [review]
LGTM

**Pull request merged**